README.md

```diff
@@ -1 +1,180 @@
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [SETUP-AND-MOBILE-FIRST] <a name=SETUP-AND-MOBILE-FIRST></a>
+
+
+**[Setup and mobile first]** is a simple project built with html and css showing responsiveness for mobile version.
+
+## 🛠 Built With
+1. Html
+2. Css
+3. Linters
+
+
+<details>
+  <ul>
+    <li><a href="https://html.org/">Html</a></li>
+    <li><a href="https://css.org/">Css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+- Added icons from bootstrap and images
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 [Live Demo](https://riskmaris.github.io/Setup-and-Mobile-First/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+```bash
+git clone https://github.com/riskmaris/Setup-and-Mobile-First.git
+cd Setup-and mobile-first
+npm install
+```
+
+### Prerequisites
+
+In order to run this project you need:
+- Web browser
+- Have visual studio code installed
+- Install node.js
+- Install git
+- You need to Go Live
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/riskmaris/Setup-and-Mobile-First.git
+```
+
+### Install
+
+Install this project with this command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Maris Kapuka
+
+- GitHub: [riskmaris](https://github.com/riskmaris)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add header
+- Add footer
+- Add styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse with all the resources they provided.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.txt) licensed<a name="readme-top"></a>
```

Comment:
- [ ] Kindly add an `MIT` License to your project. The purpose of the MIT License is to grant permission to use, copy, modify, and distribute a software project, along with any associated documentation, under certain conditions.